src/roiextractors/extractors/suite2p/suite2psegmentationextractor.py

```diff
@@ -70,7 +97,11 @@ def __init__(
         self._image_mean = self._summary_image_read("meanImg")
 
     def _load_npy(self, filename, mmap_mode=None):
```

Comment:
Though with my suggested modification, not a bad idea to improve this private functions name+annotation+docstring to reflect new behavior

```suggestion
    def _attempt_load_npy(self, filename, mmap_mode=None) -> np.ndarray | None:
    """Attempt to load the .npy file at the in the current `plane_no` subfolder of this extractor; return None if file is missing."""
    
```